### ISSUE TITLE
Maps Swagger documentation for Get Map /map wrong values

### ISSUE THREAD

**Issue by CeciliaPH** [geopython/pygeoapi#1808]
**Description**
When using Swagger to get a map, values for bbox-crs are not consistent with how pyegoapi obtains the map. Either it should be empty or allow to enter an URL if it is expected. 

I think there is something else as reprojection by crs is not working... but at least it would be nice if values in swagger examples are correct.
https://github.com/geopython/pygeoapi/issues/1648

**Steps to Reproduce**
KO
https://api-maps.idee.es/collections/IGNBaseTodo-gris/map?f=png&bbox=0.5,38,5,42.5&bbox-crs=4326 (swager example)
https://api-maps.idee.es/collections/IGNBaseTodo-gris/map?f=png&bbox=0.5,38,5,42.5&bbox-crs=http://www.opengis.net/def/crs/EPSG/0/4326

Empty image:
https://api-maps.idee.es/collections/IGNBaseTodo-gris/map?f=png&bbox=385221,4438425,483717,4507890&bbox-crs=http://www.opengis.net/def/crs/EPSG/0/25830
https://api-maps.idee.es/collections/IGNBaseTodo-gris/map?f=png&bbox=385221,4438425,483717,4507890&bbox-crs=http://www.opengis.net/def/crs/EPSG/0/25830

OK
https://api-maps.idee.es/collections/IGNBaseTodo-gris/map?f=png&bbox=0.5,38,5,42.5
https://api-maps.idee.es/collections/IGNBaseTodo-gris/map?f=png&bbox=0.5,38,5,42.5&crs=4326

**Expected behavior**
It would be great if swagger uses the right values.

**Screenshots/Tracebacks**
(https://api-maps.idee.es/openapi?f=html)

**Comment by CeciliaPH** [comment]
Hi Tom,

I was reviewing some of the historical issues and my understanding is that this is still outstanding. Our current version is v0.20.

I did a bit more analysis and it looks like the crs parameter may be missing. Another possible approach would be to make bbox-crs "with a blank value" so that Swagger can be used without issues, instead of the 4326 value that it raises as a default.

App developers make use of Swagger, and this topic keeps coming up, so addressing it would help avoid recurring questions.

Thanks for taking a look.

Best regards,
 Cecilia

**Comment by doublebyte1** [comment]
> Hi Tom,
> 
> I was reviewing some of the historical issues and my understanding is that this is still outstanding. Our current version is v0.20.
> 
> I did a bit more analysis and it looks like the crs parameter may be missing. Another possible approach would be to make bbox-crs "with a blank value" so that Swagger can be used without issues, instead of the 4326 value that it raises as a default.
> 
> App developers make use of Swagger, and this topic keeps coming up, so addressing it would help avoid recurring questions.
> 
> Thanks for taking a look.
> 
> Best regards, Cecilia

Good catch @CeciliaPH ! There were [some updates](https://github.com/geopython/pygeoapi/pull/2358) since you first wrote and now the CRS parameter needs to be in the format of an URI, a CURIE or a safe CURIE (as recommended in the Standard); so crs=4326 is not acceptable anymore.

Indeed the OpenAPI definition was out of sync with the code. It should be fixed on this PR: https://github.com/geopython/pygeoapi/pull/2406

(I used the opportunity to switch the default value from 4326 to CRS84, which is the default in the Standard)

It seems to work fine now, but let me know otherwise!

<img width="2041" height="1296" alt="Image" src="https://github.com/user-attachments/assets/1895a0bb-563e-4725-8bc7-2bb2b334697f" />

<img width="2041" height="1296" alt="Image" src="https://github.com/user-attachments/assets/ef82a2a8-bda7-4b49-abaa-7dbb5b33f6d9" />

Thanks again for keeping an eye on the API! 🥇

**Comment by CeciliaPH** [comment]
thank you all!
